components/BottomNavigation/src/MDCBottomNavigationBarController.m

```diff
@@ -27,6 +27,7 @@
 static const NSTimeInterval kLargeItemViewAnimationDuration = 0.1;
 static const NSTimeInterval kLongPressMinimumPressDuration = 0.2;
 static const NSUInteger kLongPressNumberOfTouchesRequired = 1;
+static NSString *const kSelectedViewControllerRestorationKey = @"selectedViewController";
```

Comment:
Should this key be unique to this class? Maybe something like `MDCBottomNavigationBarControllerSelectedViewControllerKey`?

Reply:
The given coder is scoped to the particular controller.  The WWDC presentation introducing state restoration says you do not have to do this.

Reply:
Awesome, thanks for clarifying!